src/pages/documentation/adobe-analytics/tabs/product-variable.md

```diff
@@ -24,7 +24,24 @@ MobileCore.trackAction("purchase", cdata);
 // trackState example:
 MobileCore.trackState("Order Confirmation", cdata);
 ```
+#### Kotlin
 
+**Example**
+
+```kotlin
+//create a context data dictionary
+val cdata: Map<String, Any?> = mapOf(
+    "&&products" to ";Running Shoes;1;69.95,;Running Socks;10;29.99",
+    "myapp.purchase" to "1",
+    "myapp.purchaseid" to "1234567890"
+)
+
+// send a tracking call - use either a trackAction or TrackState call.
```

Comment:
nit: check the casing - ... trackAction or trackState ...

and thanks for adding the kotlin samples 👍